Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Pull and use the official Docker Hub Ubuntu 14.04 base image
+FROM ubuntu:14.04
+
+# Update the base image
+RUN apt-get update
+RUN apt-get -y upgrade
+RUN apt-get -y dist-upgrade
+
+# Install Timesketch dependencies
+RUN apt-get -y install python-pip python-dev libffi-dev python-psycopg2
+
+# Install Plaso
+RUN apt-get -y install software-properties-common 
```

Comment:
Should be a single multi line RUN statement for all apt-get actions otherwise it may cause caching issues. See [dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/).